app/controllers/categories/indicators_controller.rb

```diff
@@ -79,5 +70,9 @@ def indicator_params
       def set_category
         @category = Category.find(params[:category_id])
       end
+
+      def sort_column
```

Comment:
@hengsokly this method seems duplicated almost every controller, can we move it into concern where it could plug into a controller that is sortable?